src/components/MatchBreakdownTable/breakdown2019.js

```diff
@@ -0,0 +1,351 @@
+          blueBreakdown.teleopPoints,
+        ]}
+        total={true}
+      />
+
+      <BreakdownRow
+        data={[
+          "Complete Rocket",
+          redBreakdown.completeRocketRankingPoint ? vIcon : xIcon,
+          blueBreakdown.completeRocketRankingPoint ? vIcon : xIcon,
+        ]}
+      />
+
+      <BreakdownRow
+        data={[
+          "HAB Docking",
+          redBreakdown.habDockingRankingPoint ? vIcon : xIcon,
+          blueBreakdown.habDockingRankingPoint ? vIcon : xIcon,
+        ]}
+      />
+
+      <BreakdownRow
+        data={[
+          "Fouls",
+          `+${redBreakdown.foulPoints}`,
+          `+${blueBreakdown.foulPoints}`,
+        ]}
+      />
+
+      <BreakdownRow
+        data={[
+          "Adjustments",
+          redBreakdown.adjustPoints,
+          blueBreakdown.adjustPoints,
+        ]}
+      />
+
+      <BreakdownRow
+        data={[
+          "Total Score",
+          redBreakdown.totalPoints,
+          blueBreakdown.totalPoints,
+        ]}
+        total={true}
+      />
+
+      {match.comp_level === "qm" ? (
+        <BreakdownRow
+          data={[
+            "Ranking Points",
+            `+${redBreakdown.rp} RP`,
+            `+${blueBreakdown.rp} RP`,
+          ]}
+        />
+      ) : null}
+    </div>
+  );
+};
+
+const nullHatchPanelImage = (
+  <Tooltip title="Null Hatch Panel" placement="top">
+    <TripOriginIcon style={{ fill: "#616161" }} />
+  </Tooltip>
+);
+const hatchPanelImage = (
+  <Tooltip title="Hatch Panel" placement="top">
+    <TripOriginIcon style={{ fill: "#ffeb3b" }} />
+  </Tooltip>
+);
+const cargoImage = (
+  <Tooltip title="Cargo" placement="top">
+    <LensIcon style={{ fill: "#ff6d00" }} />
+  </Tooltip>
+);
+const xIcon = <Close />;
+const vIcon = <CheckRounded />;
+
+const ImageCount = (image, count) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      {image}
+      <div
+        style={{ paddingLeft: 1, paddingRight: 4, lineHeight: 1, fontSize: 14 }}
+      >
+        {count}
+      </div>
+    </div>
+  );
+};
+
+const getSandstormBonusFor = (breakdown, robotNumber) => {
+  if (breakdown["habLineRobot" + robotNumber] === "CrossedHabLineInSandstorm") {
+    const result = breakdown["preMatchLevelRobot" + robotNumber];
+    if (result.includes("HabLevel")) {
+      const level = result.substr(-1);
+      const climbPoints = [3, 6];
+      return `Level ${level} (+${climbPoints[level - 1]})`;
+    }
+  }
+  return xIcon;
+};
+
+const getHABClimbFor = (breakdown, robotNumber) => {
+  const result = breakdown["endgameRobot" + robotNumber];
+  if (result.includes("HabLevel")) {
+    const level = result.substr(-1);
+    const climbPoints = [3, 6, 12];
+    return `Level ${level} (+${climbPoints[level - 1]})`;
+  }
+  return xIcon;
+};
+
+const getCargoShipDataFor = breakdown => {
```

Comment:
`getCargoShipDataFor(breakdown)` really should just be `<CargoShipData breakdown={breakdoown} />`, and instead of returning `[ImageCount(), ImageCount(), ImageCount()]` it should return

```
<>
  <ImageCount />
  <ImageCount />
  <ImageCount />
</>
```

Right now the console has a warning `Warning: Each child in a list should have a unique "key" prop.` because you'r returning an array.

TLDR: While React components are technically functions, composing them as components rather than function calls is preferred.

Reply:
Can you fix this? The console is throwing warnings still.